source/documentation/basic-project.md

```diff
@@ -81,6 +81,69 @@ generating the site.
 With this option passed, `{{ site.url }}/about` will now be generated as
 `http://my.dev.host/blog-skelton/output_dev/about` instead of `/about`.
 
+## CSS and JavaScript
+
+It is best practice to combine all your CSS files and all your JavaScript files
+into single files. That way they are simple to include in your templates and
+create less overhead for the browser loading the page. The recommended way to
+combine assets is with Symfony's
+[Webpack Encore](https://symfony.com/doc/current/frontend.html).
+
+The [Blog Skeleton](https://github.com/sculpin/sculpin-blog-skeleton) project
+uses Encore to manage assets. If you're setting up your project with Sculpin
+directly, not using the skeleton, install Encore using the
+[setup instructions](https://symfony.com/doc/current/frontend/encore/installation.html).
+
+The rest of this section assumes that you are using Encore, with a
+configuration like
+[in the skeleton](https://github.com/sculpin/sculpin-blog-skeleton/blob/master/webpack.config.js).
+
+Put your CSS and JS assets in the directory `source/assets/` as shown.
+
+    |-- source/
+       `-- assets/
+          |-- css/
+          |  `-- app.css
+          `-- js/
+             `-- app.js
+
+The app's main CSS file should be `source/assets/css/app.css`.
+You can add more CSS files in the same directory if necessary and require them
+in your JavaScript files to use them.
```

Comment:
why javascript files? "... and require them in your app.css file to make them included."?

Reply:
You *can* do it that way, but the preferred way is really to require them in your javascript files. (It's also possible to add CSS files to `webpack.config.js` by themselves, but again, not preferred. https://symfony.com/doc/current/frontend/encore/simple-example.html#compiling-only-a-css-file)

Reply:
so the css is embedded in javascript? isn't that bad for page loading performance? or how should i understand "require"?